### PR TITLE
Scene mgr

### PR DESCRIPTION
**Pull request by gdxn96**

### CODE REVIEW

File: spikes/sceneMgr/ConsoleApplication1.cpp [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+	GameScene game = GameScene();
+	SplashScene splash = SplashScene();
+	MainMenuScene menu = MainMenuScene();
+
+	SceneManager::getInstance()->addScene(&game);
+	SceneManager::getInstance()->addScene(&splash);
+	SceneManager::getInstance()->addScene(&menu);
+	SceneManager::getInstance()->switchTo(Scenes::GAME);
+	SceneManager::getInstance()->switchTo(Scenes::MAINMENU);
+	SceneManager::getInstance()->switchTo(Scenes::SPLASH);
```
Comment (ShaneCallanan) [inline spikes/sceneMgr/ConsoleApplication1.cpp:53]: Why switch to 3 different scenes at beginning?

File: spikes/sceneMgr/ConsoleApplication1.cpp [hunk 2]
```diff
@@ -0,0 +1,100 @@
+////////////////////////////////////////////////////////////
```
Comment (ShaneCallanan) [inline spikes/sceneMgr/ConsoleApplication1.cpp:1]: Coding standards (80char)

File: spikes/sceneMgr/Scene.cpp [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+void Scene::enter()
+{
+	alive = true;
+}
+
+void Scene::exit()
+{
+	alive = false;
+}
```
Comment (ShaneCallanan) [inline spikes/sceneMgr/Scene.cpp]: enter() and exit() no longer in use

File: spikes/sceneMgr/ConsoleApplication1.cpp [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+			// Close window : exit 
+			if (Event.type == sf::Event::Closed)
+				window.close();
+
+			// Escape key : exit 
+			if ((Event.type == sf::Event::KeyPressed) && (Event.key.code == sf::Keyboard::Escape))
+				window.close();
+
+			/////////////////////////////////////////////////////////////////////////////////////////////////
+			if (sf::Keyboard::isKeyPressed(sf::Keyboard::G))
```
Comment (Lanc3) [inline spikes/sceneMgr/ConsoleApplication1.cpp:71]: we will need to do menu assets to navigate the menu the right way. using the input handler

**Pull request merged**